DhawaAlkhreej/DhawaAlkhreej/Resources/Resources.swift

```diff
@@ -21,9 +21,43 @@ struct Resources {
         }
         
         var twitter: String {
-            return "cs_alhaider"
+            return "https://twitter.com/cs_alhaider"
```

Comment:
We can use only the username than pass it to the url like: 
` let twitterURL = "http://twitter.com/\(twitter)"`